bluesky/callbacks/broker.py

```diff
@@ -33,24 +58,16 @@ def __init__(self, field, *, fs=None, cmap=None, norm=None,
                  limit_func=None, auto_redraw=True, interpolation=None):
         from xray_vision.backend.mpl.cross_section_2d import CrossSection
         import matplotlib.pyplot as plt
-        super().__init__()
-        self.field = field
```

Comment:
I think it's useful for `LiveImage` to know its own field. I think it is clearer than referring to `self.fields[0]` of its parent.